source/fundamentals/atlas-search.txt

```diff
@@ -142,21 +138,69 @@ The search returns the following documents:
 
 .. code-block:: json
 
-   { "_id" : 1, "make" : "Fender", "description" : "...", "establishedYear" : 1946, "in-stock" : true, "rating" : 9 }
-   { "_id" : 3, "make" : "PRS", "description" : "...", "establishedYear" : 1985, "in-stock" : true, "rating" : 9 }
-   { "_id" : 5, "make" : "Ibanez", "description" : "...", "establishedYear" : 1957, "in-stock" : true, "rating" : 7 }
+   { "_id" : 1, "make" : "Fender", "description" : "...", "establishedYear" : 1946, "in_stock" : true, "rating" : 9 }
+   { "_id" : 3, "make" : "PRS", "description" : "...", "establishedYear" : 1985, "in_stock" : true, "rating" : 9 }
+   { "_id" : 5, "make" : "Ibanez", "description" : "...", "establishedYear" : 1957, "in_stock" : true, "rating" : 7 }
 
 To learn more about the ``compound`` operator, see the :atlas:`compound </atlas-search/compound>` 
 Atlas guide.
 
+Embedded Document
```

Comment:
Nit: Following the pattern for the rest of the section headers, this should be one word in UpperCamelCase.

Reply:
Good catch!